backend/app/services/readers/kinds.py

```
@@ -0,0 +1,271 @@
+class KindReader(IKindReader):
+    """Kind reader with direct database queries."""
+
+    def get_by_id(
+        self, db: Session, kind: KindType, resource_id: int
+    ) -> Optional[Kind]:
+        return (
+            db.query(Kind)
+            .filter(
+                Kind.id == resource_id,
+                Kind.kind == kind.value,
+                Kind.is_active == True,
```

Ruff (0.14.10)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)